app/gimlet-rot/app-b.toml

```diff
@@ -21,6 +21,9 @@ features = ["itm"]
 stacksize = 1536
 notifications = ["fault", "timer"]
 
+[tasks.jefe.config.allowed-callers]
+request_reset = ["hiffy", "update_server"]
```

Comment:
Why `hiffy` here?

Reply:
Good point, `hiffy` is not calling directly to jefe, but going through `update_server`. I believe this can be removed here and in the other `.toml` files. will do.